certs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
This is so that if one desired, one can build and run this image only using `docker-compose`.  With `docker-compose`, you can provide runtime arguments to docker, removing the need for a complicated `docker run` statement.  `docker-compose` needs somewhere to put the cert when obtained, and I thought a relative path to be most appropriate, making the certs more portable.  From what I've learned since, `Let's Encrypt` tends to rely on long-standing authentication keys that are stored in `/etc/letsencrypt`, so it is less clear to me now that the quick "request-and-dump-cert" that I'm doing here is all that helpful.

In the future, it would make more sense to have docker in a separate container but have access to a neighboring container that is running `apache` (or whatever software is using the cert) via the `--volumes-from` flag.  This would expose the necesary paths for Let's Encrypt to rewrite the config files it needs to.